http/resp/responder.go

```diff
@@ -409,6 +418,49 @@ func (doer *Responder) do(w http.ResponseWriter, r *http.Request, opts ...Fn) (*
 	return resp, nil
 }
 
+// handleHtmlError specially renders the error template set on the Responder
+// and reports errors.
+func (doer *Responder) handleHtmlError(w http.ResponseWriter, r *http.Request, err error) error {
+	w.WriteHeader(http.StatusInternalServerError)
+
+	if doer.templates.err == "" {
+		err = fmt.Errorf(
+			"%w: no error template provided, encountered while handling: %s",
+			ErrBadConfig,
+			err,
+		)
+		return err
+	}
+	b := doer.pool.Get().(*bytes.Buffer)
+	b.Reset()
+	defer doer.pool.Put(b)
+
+	tmpl, nested := doer.parser.Parse(doer.templates.err)
+	if nested != nil {
+		err = fmt.Errorf("%w: %s", nested, err)
+		doer.logger.Error(err.Error(), nil)
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return err
+	}
+
+	nested = tmpl.Execute(b, map[string]any{"Contact": doer.contactErrMsg, "Error": err})
```

Comment:
I guess this is a spot where contactErrMsg could be empty. This is where config would step in to help make it clear what is required for ranger to really hum

Reply:
Yeah definitely, would love to see "webmaster" contact and a default flash message in `ranger.Config` to ensure these values are available deeper down.